docs/make.jl

```diff
@@ -16,6 +16,7 @@ makedocs(
         "mixture.md",
         "fit.md",
         "extends.md",
+        "contributing.md"
```

Comment:
```suggestion
        "contributing.md",
```
So that adding new lines reduces the diff